Sources/Amplitude/AMPURLSession.m

```diff
@@ -59,29 +59,42 @@ + (AMPURLSession *)sharedSession {
 
 - (instancetype)init {
     if ((self = [super init])) {
-        [AMPURLSession pinSSLCertificate:@[@"ComodoRsaDomainValidationCA"]];
+        // ComodoRsaDomainValidationCA is for US endpoint, AmazonRootCA1 is for EU endpoint
+        NSDictionary *domainCertFilenamesMap = @{
+            kAMPEventLogDomain: @[
+                @"ComodoRsaDomainValidationCA.der",
+                @"Amplitude_Amplitude.bundle/ComodoRsaDomainValidationCA.der"
+            ],
+            kAMPEventLogEuDomain: @[
+                @"AmazonRootCA1.cer",
+                @"Amplitude_Amplitude.bundle/AmazonRootCA1.cer"
+            ]
+        };
+        [AMPURLSession pinSSLCertificate:domainCertFilenamesMap];
         NSURLSessionConfiguration *configuration = [NSURLSessionConfiguration defaultSessionConfiguration];
         _sharedSession = [NSURLSession sessionWithConfiguration:configuration delegate:self delegateQueue:nil];
     }
     return self;
 }
 
-+ (void)pinSSLCertificate:(NSArray *)certFilenames {
++ (void)pinSSLCertificate:(NSDictionary *)domainCertFilenamesMap {
     // We pin the anchor/CA certificates
-    NSMutableArray *certs = [NSMutableArray array];
-    for (NSString *certFilename in certFilenames) {
-        NSString *certPath =  [[NSBundle bundleForClass:[self class]] pathForResource:certFilename ofType:@"der"];
-        NSData *certData = [[NSData alloc] initWithContentsOfFile:certPath];
-        if (certData == nil) {
-            AMPLITUDE_LOG(@"Failed to load a certificate");
-            return;
+    NSMutableDictionary *pins = [[NSMutableDictionary alloc] init];
+    for (id domain in domainCertFilenamesMap) {
+        NSMutableArray *certs = [NSMutableArray array];
+        NSArray *certFilenames = [domainCertFilenamesMap objectForKey:domain];
+        for (NSString *certFilename in certFilenames) {
+            NSString *certPath =  [[NSBundle bundleForClass:[self class]] pathForResource:certFilename ofType:nil];
+            NSData *certData = [[NSData alloc] initWithContentsOfFile:certPath];
+            if (certData == nil) {
```

Comment:
Question here:
The workaround you are using for the `der` file, is you define 2 paths for each certificate.  The `der` file might only exist in one path.
Does that mean it will fall into this condition and print out the `failed to load certificate` log, even if we can get the certificate from 1 path?

Reply:
Good question. I thought about this, my understand is this log is more for debugging purpose for us/experienced customer, as we cannot figure out the exact path for the file, better to output more when trouble happens, as SDK users don't need to know the existence of these certs for this function. Let me know if you have any suggestions on this.

Reply:
Can we have another loop after we finished the `[pins setObject:certs forKey:domain]`. 
And we can check if it has certs is nill for all domains.